src/core/tests/Test_CoreMain.py

```diff
@@ -1143,6 +1168,624 @@ def __check_telemetry_events(self, runtime):
+        patch_count = random.randint(200, 432)
+        test_packages, test_package_versions = self.__set_up_packages_func(patch_count)
+        runtime.status_handler.set_package_assessment_status(test_packages, test_package_versions)
+        runtime.status_handler.set_assessment_substatus_json(status=Constants.STATUS_SUCCESS)
+
+        # Assert complete status file
+        with runtime.env_layer.file_system.open(runtime.execution_config.complete_status_file_path, 'r') as file_handle:
+            substatus_file_data = json.load(file_handle)
+
+        self.assertTrue(len(json.dumps(substatus_file_data)) < Constants.StatusTruncationConfig.AGENT_FACING_STATUS_FILE_SIZE_LIMIT_IN_BYTES)
+        self.__assert_complete_status_output(substatus_file_data, Constants.ASSESSMENT, Constants.PATCH_ASSESSMENT_SUMMARY, Constants.STATUS_SUCCESS, patch_count + 3)
+
+        # Assert truncated status file
+        with runtime.env_layer.file_system.open(runtime.execution_config.status_file_path, 'r') as file_handle:
+            substatus_file_data = json.load(file_handle)
+
+        self.assertTrue(len(json.dumps(substatus_file_data)) < Constants.StatusTruncationConfig.AGENT_FACING_STATUS_FILE_SIZE_LIMIT_IN_BYTES)
+        substatus_file_data = substatus_file_data[0]["status"]["substatus"][0]
+        self.assertEqual(substatus_file_data["name"], Constants.PATCH_ASSESSMENT_SUMMARY)
+        self.assertNotEqual(substatus_file_data["status"], Constants.STATUS_WARNING.lower())
+        self.assertEqual(len(json.loads(substatus_file_data["formattedMessage"]["message"])["patches"]), patch_count + 3)
+        # status_file_patches = json.loads(substatus_file_data["formattedMessage"]["message"])["patches"]
+        # self.assertNotEqual(status_file_patches[1]['patchId'], "Truncated_patch_list_id")
+        # self.assertTrue('additional updates of classification' not in status_file_patches[1]['name'])
+        self.assertEqual(json.loads(substatus_file_data["formattedMessage"]["message"])["errors"]["code"], 0)
+        self.assertEqual(len(json.loads(substatus_file_data["formattedMessage"]["message"])["errors"]["details"]), 0)
+        self.assertFalse("review this log file on the machine" in json.loads(substatus_file_data["formattedMessage"]["message"])["errors"]["message"])
+        # self.assertEqual(len(runtime.status_handler._StatusHandler__assessment_packages_removed), 0)
+
+        runtime.stop()
+
+    def test_assessment_operation_truncation_over_size_limit(self):
+        argument_composer = ArgumentComposer()
+        argument_composer.operation = Constants.ASSESSMENT
+        runtime = RuntimeCompositor(argument_composer.get_composed_arguments(), True, Constants.ZYPPER)
+        runtime.set_legacy_test_type('HappyPath')
+        CoreMain(argument_composer.get_composed_arguments())
+
+        # Check telemetry events
+        self.__check_telemetry_events(runtime)
+
+        # HappyPath add 3 additional packages
```

Comment:
HappyPath doesn't add additional packages, we are adding additional packages on lines 1231-1234.

See my comment in the previous function and update this accordingly

Reply:
done

Reply:
Code does not exist now